### PR TITLE
Adding more details

### PR DESCRIPTION
**Pull request by nxl365**
write 'Nancy' in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # testing_prs
 Testing and showing how PR:s work.
+
+test_writed by Nancy
```
Comment (ljmesi) [inline README.md:4]:
```suggestion
test_writed by Nancy
```
Should this have capital letter, like `Test_writed`.

Comment (ljmesi) [inline README.md:4]:
```suggestion
Test_writed by Nancy
```
How about this way?

**Review by ljmesi — COMMENTED** [review]
Looks really good! Good job!

File: README.md (hunk 1 above)
Comment (nxl365) [reply README.md]: good!

**Review by ljmesi — APPROVED** [review]
Really good job Nanxing!

**Pull request merged**